test/csv2rdf/csvw_test.clj

```diff
@@ -6,28 +6,37 @@
             [csv2rdf.csvw :as csvw]
             [csv2rdf.http :as http]
             [grafter-2.rdf4j.io :as gio])
-  (:import [java.net URI URL]))
-
-(deftest
-  ^
-  {:title "Simple table", :description "The simplest possible table without metadata"}
-  test001
-  (let [http-client (->TestHttpClient
-                      {(URI. "http://www.w3.org/2013/csvw/tests/test001.csv")
-                       {:body (io/file "w3c-csvw/tests/test001.csv"),
-                        :headers {"Content-Type" "text/csv; charset=UTF-8"}}})
-        csv-uri (URI. "http://www.w3.org/2013/csvw/tests/test001.csv")
-        metadata-uri nil
-        {:keys [warnings errors result]} (http/with-http-client
-                                           http-client
-                                           (test-csv->rdf csv-uri metadata-uri {:mode :standard}))]
-    (let [expected-statements (gio/statements
-                                (io/file "w3c-csvw/tests/test001.ttl")
-                                :base-uri
-                                (URI. "http://www.w3.org/2013/csvw/tests/test001.csv"))]
-      (is (= true (is-isomorphic? expected-statements result))))
-    (is (= 0 (count warnings)) "Received warnings but none was expected")
-    (is (= 0 (count errors)) "Received errors but none was expected")))
+  (:import [java.net URI]
+           [java.nio.file FileSystems]
+           [grafter_2.rdf.protocols Quad]))
+
+(defn- make-path [& path]
+  (.getPath (FileSystems/getDefault) "w3c-csvw" (into-array String path)))
+
+(deftest csv->rdf-with-paths
```

Comment:
Could we move this test into a different file - the `csvw_test.clj` file is generated from the test `manifest.csv` file. See `csvw-test.gen/write-tests-file`.

Reply:
yeah I did consider putting it in a different file, as I knew they'd been generated... just wasn't sure exactly how.